### PR TITLE
 Added support for quetre

### PR DESCRIPTION
**Pull request by tomit4**
To Sudeep Biswas,

Thank you for creating the useful browser extension, alter! I have found it very useful.

I have created this pull request because I have managed to remove Bibliogram, which, unfortunately, is now defunct and no longer available. I have also added support for [quetre](https://codeberg.org/zyachel/quetre), a privacy respecting Front End for Quora. 

If you could please review and merge these changes. I believe that integrating quetre with alter is in line with the project's goals.

Should you have any questions or concerns, please feel free to reach out to me.

Thank you for your time and consideration.

Sincerely,
Brian Hayes

### CODE REVIEW

File: popup.html [hunk 1]
```diff
@@ -43,23 +43,19 @@
...
       <fieldset>
-        <legend>Scribe instances</legend>
-        <select id="scribe-instances" name="scribe-instances"></select>
-        <input name="scribe-instance" type="text" placeholder="Custom domain" />
+        <legend>Quetre instances</legend>
+        <select id="quetre-instances" name="quetre-instances"></select>
+        <input name="quetre-instance" type="text" placeholder="Custom domain" />
         <button class="add-custom" type="button">+</button>
         <label class="switch">
           <input id="switch-scribe" type="checkbox" checked />
```
Comment (w3bdev1) [inline popup.html]:
```suggestion
          <input id="switch-quetre" type="checkbox" checked />
```

File: popup.js [hunk 2]
```diff
@@ -169,13 +169,12 @@ document.forms[0].onsubmit = (e) => {
     teddit: getInputValue(data.teddit.el),
     invidious: getInputValue(data.invidious.el),
     scribe: getInputValue(data.scribe.el),
-    bibliogram: getInputValue(data.bibliogram.el),
```
Comment (w3bdev1) [inline popup.js:172]: Current form value of quetre instance is missing

```
    quetre: getInputValue(data.quetre.el),
```

**Comment by w3bdev1** [comment]
Apart from the above 2 issues, I would suggest you to put other instances for Quetre as the current ones given are either mostly blocked (quetre.iket.me) or not responding (quetre.vern.cc).

File: popup.html (hunk 1 above)
Comment (tomit4) [reply popup.html]: Ok, Thanks for catching that. Sorry about such a simple miss.

File: popup.js (hunk 2 above)
Comment (tomit4) [reply popup.js:172]: Ah, and another simple miss. I did notice some strange behavior due to this. Thanks again.

**Comment by tomit4** [comment]
I placed other instances available in a commented out line on my recent commit. The reason I left them commented out was because I only experienced only the occasional server side error from the various quetre instances, which I think might have to due with some form of rate limiting, although I'm not sure. If you'd like to see which quetre instances work on your end, you can play around with some of the mirrors listed on their codeberg repo:

https://codeberg.org/zyachel/quetre/#instances

Again, thank you.

**Comment by tomit4** [comment]
I have now also added a first draft of adding openstreetmaps redirect option. It is not perfect. You were right, due to the interesting query strings generated by both google and openstreet map, getting it to work where you can simply use duckduckgo bangs and type in directions into gmaps and then have it redirect to openstreetmaps is tricky because the url string is lengthy and even though there is obvious use of latitude and longitude, where and how their respective APIs create that url is different enough that I'm not sure if it can be solved by a simple regex check and replace...

Nevertheless this more or less works for single address queries. It does add a forward slash '/' character to the beginning of the query on openstreetmaps. I'm not sure why this is, but perhaps you could take a look at it and figure it out.

Hopefully this is helpful. Let me know what you think.

**Comment by w3bdev1** [comment]
OSM redirection is a different beast altogether. Kindly put it in a separate PR. We can discuss it there. Thank you :)

**Pull request merged**